src/editor/icons/icons.jsx

```diff
@@ -757,6 +757,77 @@ const SignInMicrosoftIconSVG = () => (
   </svg>
 );
 
+const ManagedStreetIcon = () => (
```

Comment:
Really in this file we should use `export const ManagedStreetIcon` instead of adding a line at the bottom of the file each time we add an icon.

Reply:
bdcc04f89b35d76d0349b4da0ca4c4b52365ba7f